js_modules/dagster-ui/packages/ui-core/src/app/AppError.tsx

```diff
@@ -218,28 +216,21 @@ export const setupErrorToasts = () => {
         // will trigger the "Can't re-render component during render" console error
         // which would send us in an infinite loop. So we use setTimeout to avoid this.
         setTimeout(async () => {
-          const toaster = await getErrorToaster();
-          toaster.show({
-            intent: 'danger',
-            message: (
-              <div
-                style={{whiteSpace: 'pre-wrap', maxHeight: 400, overflow: 'hidden'}}
-              >{`console.error: ${msg}`}</div>
-            ),
-          });
+          showErrorToast(
+            <div
+              style={{whiteSpace: 'pre-wrap', maxHeight: 400, overflow: 'hidden'}}
+            >{`console.error: ${msg}`}</div>,
+          );
         }, 0);
       }
     },
   });
 
   window.addEventListener('unhandledrejection', async (event) => {
-    (await getErrorToaster()).show({
-      intent: 'danger',
-      message: (
-        <div
-          style={{whiteSpace: 'pre-wrap'}}
-        >{`Unhandled Rejection: ${event.reason}\nView console for details.`}</div>
-      ),
-    });
+    showErrorToast(
+      <div
+        style={{whiteSpace: 'pre-wrap'}}
+      >{`Unhandled Rejection: ${event.reason}\nView console for details.`}</div>,
```

Comment:
Calling out this one because it has a newline in it, might be nice to verify it looks ok